R/fallback.R

```diff
@@ -141,19 +133,6 @@ fallback_txt_help <- function() {
   )
 }
 
-fallback_nudge <- function(call_data) {
```

Comment:
Why are we losing this?

Reply:
to simplify. We advertise in other places and if the docs are clearer (they are not clearer yet) then no nudge is needed.